packages/firebase/index.d.ts

```diff
@@ -154,10 +154,10 @@ declare namespace firebase.auth {
     applyActionCode(code: string): Promise<any>;
     checkActionCode(code: string): Promise<any>;
     confirmPasswordReset(code: string, newPassword: string): Promise<any>;
-    createUserAndRetrieveDataWithEmailAndPassword( 
-      email: string, 
-      password: string 
-    ): Promise<any>;      
+    createUserAndRetrieveDataWithEmailAndPassword(
```

Comment:
@jshcrowthe this prettier auto stuff really drags on PRs :(

Reply:
Agreed,  @schmidt-sebastian and I have both proposed solutions to handle this (#585, #580). Both scope the change down to only your package so other components changes shouldn't be included as much.